File: Kotlin/Leetcode0009.kt
class Leetcode0009 {
    fun isPalindrome(x: Int): Boolean {
        val number = Integer.toString(x)
        var answer = true
        val length = number.length
        for(i in 0..length / 2) {
            if((number[i]) != number[(length - 1) - i]) {
                answer = false
            }
        }
        return answer
    }
}

File: Go/Leetcode0009.go
package main

import (
	"fmt"
	"strconv"
)

func l0009s() {
	fmt.Print(isPalindrome(1221))
}

func isPalindrome(x int) bool {
	number := strconv.Itoa(x)
	for i := 0; i < (len(number) / 2); i++ {
		if (string(number[i])) != string(number[(len(number)-1)-i]) {
			return false
		}
	}
	return true
}
